mozetl/taar/taar_locale.py

```diff
@@ -65,6 +65,11 @@ def get_addons(spark):
           AND value['app_disabled'] = FALSE -- exclude compatibility disabled addons
           AND value['is_system'] = FALSE -- exclude system addons
           AND locality <> 'null'
+          AND key <> 'loop@mozilla.org'
```

Comment:
You could use `key NOT IN ('loop@mozilla.org', 'firefox@getpocket.com', ...)`, to make it clear what the blacklist looks like.